client/src/components/molecules/AgendaDetail.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Box, Text } from "@/components/atoms";
+
+interface Agenda {
+  title: string;
+  subtitle: string;
+  content: string;
+}
+
+interface Props {
+  agenda: Agenda;
+}
```

Comment:
`Agenda`라는 타입은 이미 다른 스키마로 정의되어 있어서 같은 네이밍을 쓰면서 여기에서 이렇게 다른 타입으로 정의하는 것은 혼란스러울 수 있을 것 같아요.

`AgendaFoldedText.tsx` 파일에도 동일한 이름으로 다르게 타입이 정의되어 있어서 더더욱 이런 상황은 피하는게 좋을 것 같아요!

Reply:
이 케이스의 경우에는 agenda의 `title`, `subtitle`, `content` 정보만 필요한 것이라면 새로운 `Agenda` 타입을 정의하는 것보다는 각각의 필드로 나누어서 받는 것이 explicit해서 좋을 것 같아요.

컴포넌트를 사용하는 쪽에서 생각해보면 `agenda={agenda}`와 같이 prop을 넘겨주려 할 때 서로 다른 `Agenda` 타입을 가지고 있도록 정의하면 헷갈릴 것 같네요

```suggestion
interface Props {
  title: string;
  subtitle: string;
  content: string;
}
```

Reply:
맞아요. 지금 대부분의 컴포넌트에 있는 인터페이스는 테스트용 코드라서 로직 작업할때 프론트에서 가져다쓰기 편한 형태로 함께 고치기로 했습니당.